main.tf

```
@@ -1,5 +1,10 @@
 data "aws_region" "default" {}
 
+resource "random_string" "suffix" {
```

Where is this used

---

Oh i see it under takeover suffix. Can we rename this so it's clear it's only for takeover?

```suggestion
resource "random_string" "takeover_suffix" {
```